tasks/WhereIsThis.tex

```diff
@@ -32,28 +31,18 @@ \subsection*{Additional rules and remarks}
 		\item \textbf{Follower's assistance:} Helping the robot while guiding a guest (e.g.~waving, shouting) causes a score reduction of 50pts.
 	\end{itemize}
 
-	\item \textbf{Information Point:} The robot meets guests and gives directions at the \emph{Information Point}.
-	After guiding a person to its destination, it must return to the information point.
+	\item \textbf{Information Point:} The robot meets guests and gives directions at the \emph{Information Point} and then guides them to the location. The information point is not known beforehand. Right before a run at the entrance door, the team is told a position where they should start the robot.
+	After guiding a person, it must return to the information point.
 
 	\item \textbf{Naive Operators:} Optionally, questions can be asked by a \emph{Naive Operator}, i.e.~a person from the audience with no background on robotics. \emph{Naive Operators} are allowed to rephrase commands given by the referees.
 	\\\textbf{Remark:} Referees are not allowed to instruct naive operators on how to operate the robot. \textbf{Teams attempting to instruct or bias the operator will be disqualified}.
 
 	\item \textbf{Custom operator fallback:} If the robot consistently fails to understand the naive/professional operator (e.g.~3 times or more), teams can default to a custom operator.
 
-	\item \textbf{Data Recording:} Only when using Naive Operators, a team can get an additional scoring bonus by providing the recording and transcript of the issued commands.
```

Comment:
Why do we remove data recording? This must be kept as it is mentioned in the Scoresheet.

Reply:
@swachsmu brought up a good point in recording data of naive operators: we would need authorization from them (forms galore), and we would need to ensure that they would be able to request at any time for the removal of their recorded data. This also applies for all other tasks that involve naive operators (as mentioned in my notes).

To be consistent, I also removed it from the score sheet.

Reply:
However, I'm open in writing a section of recording data of only participants (as they already filled out a form of the use of their voice and image as part of RoboCup activities).